magazine.c

```diff
@@ -0,0 +1,47 @@
+/*
+  Not part of Grbl. Written by KeyMe.
+*/
+
+#include "system.h"
+#include "settings.h"
+#include "magazine.h"
+#include "limits.h"
+#include "protocol.h"
+#include "stepper.h"
+
+void magazine_init(){
+  MAGAZINE_ALIGNMENT_DDR &= ~(MAGAZINE_ALIGNMENT_MASK); // Configure as input pin
+  MAGAZINE_ALIGNMENT_PORT |= MAGAZINE_ALIGNMENT_MASK; // Enable internal pull-up resistors. Normal high operation
+  memcpy(sys.probe_position, sys.position, sizeof(int32_t)*N_AXIS); // Set the magazine alignment position to the current position
```

Comment:
If it were bracing style, capitalization, or indentation, I'd agree, but whitespace and parens around mathematical operators are typically the one place I think it always pays to be conservative and include them.  I understand your point about upstream inclusion, but:

1) We still have to be the active maintainers of this branch of the code as it is a core part of our product.  Increased readability in our code has no down side and can be very helpful in tracking down weird operator precedence and math bugs.
-and-
2) Upstream inclusion isn't guaranteed even if we did submit a PR, and they don't seem to have a clear indication of actual expected coding style  (via a contributing document or a how-to).  If there were complaints from the upstream maintainers, they would be easy to address (or we could discuss a codebase-wide refactor to help readability).